scripts/runMainnet.sh

```diff
@@ -55,7 +55,7 @@ echo "REDIS_URL='redis://127.0.0.1:6379'" >> ${app_dir}/.env
 # Supported token settings
 echo RELAYER_TOKENS=\'[\"0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2\", \"0x6033F7f88332B8db6ad452B7C6D5bB643990aE3f\", \"0xdAC17F958D2ee523a2206206994597C13D831ec7\"]\'  >> ${app_dir}/.env
 echo MIN_DEPOSIT_CONFIRMATIONS=\'{\"5000\": { \"1\": 5, \"1135\": 10 }, \"2000\": { \"1\": 4, \"1135\": 10 }, \"100\": { \"1\": 3, \"1135\": 10 } }\' >> ${app_dir}/.env
-echo RELAYER_INVENTORY_CONFIG=\'{\"wrapEtherTarget\": 0.1, \"wrapEtherThreshold\": 0.125, \"wrapEtherTargetPerChain\": { \"1\": 1 },   \"wrapEtherThresholdPerChain\": { \"1\": 2 }, \"tokenConfig\": {\"WETH\": {\"1\": { \"targetPct\": 100, \"thresholdPct\": 100, \"unwrapWethThreshold\": 0.5, \"unwrapWethTarget\": 1 },\"1135\": { \"targetPct\": 30, \"thresholdPct\": 10, \"unwrapWethThreshold\": 0.025, \"unwrapWethTarget\": 0.1, \"targetOverageBuffer\": 1.5 }}} }\' >> ${app_dir}/.env
+echo RELAYER_INVENTORY_CONFIG=\'{ \"wrapEtherTarget\": 0.1, \"wrapEtherThreshold\": 0.125, \"wrapEtherTargetPerChain\": { \"1\": 1 }, \"wrapEtherThresholdPerChain\": { \"1\": 2 }, \"tokenConfig\": { \"WETH\": { \"1\": { \"targetPct\": 100, \"thresholdPct\": 100, \"unwrapWethThreshold\": 0.5, \"unwrapWethTarget\": 1 }, \"1135\": { \"targetPct\": 30, \"thresholdPct\": 10, \"unwrapWethThreshold\": 0.025, \"unwrapWethTarget\": 0.1, \"targetOverageBuffer\": 1.5 }}, \"LSK\": { \"targetPct\": 30, \"thresholdPct\": 10, \"targetOverageBuffer\": 1.75 }, \"USDT\": { \"targetPct\": 30, \"thresholdPct\": 10 }}}\' >> ${app_dir}/.env
```

Comment:
Should we also set the same `targetOverageBuffer` for USDT?

Reply:
Across team has confirmed that `targetOverageBuffer` if not specified defaults to `1.5` but to make it explicit, I've set it for USDT.